.github/workflows/nightly.yml

```diff
@@ -0,0 +1,22 @@
+name: Update commit hash
```

Comment:
does this work? can't https://github.com/flathub/flatpak-external-data-checker be used here?

Reply:
It works on Flycast, to be confirmed with this project.
As @barthalion [said](https://github.com/flathub/flathub/issues/1582#issuecomment-826522418), it likely better to have a simple, independent CRON for projects. The commit hash is supposed to change frequently.

Reply:
It is also not well documented, for instance a `x-checker-data` comment is needed in the file to make it work. I prefer to have something simple, in control and independent.